packit/distgit.py

```diff
@@ -271,7 +270,8 @@ def upload_to_lookaside_cache(self, archive_path: str) -> None:
             raise PackitException(ex)
 
     def is_archive_in_lookaside_cache(self, archive_path: str) -> bool:
-        archive_name = os.path.basename(archive_path)
+        path = Path(archive_path)
+        archive_name = path.name
```

Comment:
It would be nice to change the `method` input type instead to not need to convert from/to `Path` everywhere.

Ideally, we would need to convert from/to `str` only on the perimeter of our code.

When we have one code that calls some function with the path argument, we want to:
- change the code of sender to send `Path` instead of `str`
- change the code of receiver to work with `Path` instead of `str`

Reply:
done!